blacklist/build.gradle

```diff
@@ -26,11 +30,12 @@ buildscript {
 }
 
 repositories {
-    mavenLocal()
-    jcenter()
-    mavenCentral()
-    maven { url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-releases' }
-    maven { url 'https://jitpack.io' }
+        mavenLocal()
+        jcenter()
+        mavenCentral()
+        maven { url 'https://jitpack.io' }
+        maven { url 'https://repo.gradle.org/gradle/libs-releases' }
+        maven { url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda' }
 }
```

Comment:
Yuk, inconsistent indentation.
But more importantly, why has this extra repo been added?
```
maven { url 'https://repo.gradle.org/gradle/libs-releases' }
```